### ISSUE TITLE
Basic authentication is no longer working in stand alone app

### ISSUE THREAD

**Issue by rfreas** [jarrodek/ChromeRestClient#651]
Previously, the plugin inside the browser would prompt for credentials using basic authentication.  This is no longer happening with the stand-alone application.  Also, the url format of https://user:password@url doesn't work either.

**Comment by jarrodek** [comment]
Hi,

Thank you for issue report.
The urls like `http://username:password@example.com/` should be handled by the app. Basically it takes username and password part from the URL and set up basic authentication header.

For now as a workaround just remove username and password from the URL and you'll be prompted for username and password after 401 status. Also, you can set up authorization header in the headers panel. You have a dialog that helps you with data encoding.

**Comment by rfreas** [comment]
Initially I tried it without the login/password in the URL, but I didn't receive a prompt to authenticate.  The server just returned an no auth error and I never received a prompt.  In the old plugin that was hosted in the Chrome browser, I *did* receive a prompt.  But in the stand alone app version, I did not.

If I am supposed to receive a prompt, then maybe something else is amiss on my local system.

Thanks for the update!
